project2/screens/movie.js

```diff
@@ -1,174 +1,59 @@
 import React from "react";
-import { Image, Text, StyleSheet, ScrollView, View } from "react-native";
-import Constants from 'expo-constants'
+import { View } from "react-native";
+import { useNavigationParam } from "react-navigation-hooks";
 import { getMovieDetails } from "../API";
-
-
-export default class movie extends React.Component {
-    static navigationOptions = {
-        title: "Movie Details",
-        headerTitleStyle: { color: "#AAAAAF", }
-    };
-
-    state = {
-        movieID: "",
-        actors: "",
-        awards: "",
-        boxOffice: "",
-        country: "",
-        dvd: "",
-        director: "",
-        genre: "",
-        language: "",
-        metascore: "",
-        plot: "",
-        poster: "",
-        production: "",
-        rated: "",
-        ratings: [],
-        released: "",
-        runtime: "",
-        title: "",
-        type: "",
-        website: "",
-        writer: "",
-        year: "",
-        imdbID: "",
-        imdbRating: "",
-        imdbVotes: "",
+import MovieInfo from "./Movie/Info";
+import MoviePoster from "./Movie/Poster";
+import s from "../Styles";
+
+const initialMovieState = {
+  movieID: "",
+  Actors: "",
+  Awards: "",
+  BoxOffice: "",
+  Country: "",
+  DVD: "",
+  Director: "",
+  Genre: "",
+  Language: "",
+  Metascore: "",
+  Plot: "",
+  Poster: "",
+  Production: "",
+  Rated: "",
+  Ratings: [],
+  Released: "",
+  Response: "",
+  Runtime: "",
+  Title: "",
+  Type: "",
+  Website: "",
+  Writer: "",
+  Year: "",
+  imdbID: "",
+  imdbRating: "",
+  imdbVotes: "",
+};
+
+const Movie = () => {
+  const id = useNavigationParam("id");
+  const [movieDetails, setMovieDetails] = React.useState(initialMovieState);
```

Comment:
Directly equivalent to `this.state` and `this.setState` - `useState` takes one argument (an initial state), and returns a tuple of the current state + a setter function. I change the state within the function like `setMovieDetails("new state here")`, pretty simple.